README.md

```diff
@@ -64,12 +64,17 @@ The easiest way to get started with Opik is by creating a free Comet account at
 
 
 
-If you'd like to self-host Opik, you create a simple local version of Opik using::
+If you'd like to self-host Opik, you can do so by cloning the repository and starting the platform using Docker Compose:
 
 ```bash
-pip install opik-installer
+# Clone the Opik repository
+git clone https://github.com/comet-ml/opik.git
 
-opik-server install
+# Navigate to the opik/deployment/docker-compose directory
+cd opik/deployment/docker-compose
+
+# Start the Opik platform
+docker compose up --detach
```

Comment:
@jacques-comet lets add another step (go to your browser to `http://localhost:5173/` since people wont know where to go after starting ?